conformance/lang/expressions/binary-bitwise-expression/binary_bitwise_and_expr.balt

```diff
@@ -42,9 +42,9 @@ function init() {
     io:println(6 & -9); // @output 6
     io:println(-43554 & -6); // @output -43558
 
-    io:println(-9223372036854775808 & -9223372036854775808); // @output -9223372036854775808
-    io:println(-9223372036854775808 & 3); // @output 0
-    io:println(-9223372036854775808 & 9223372036854775807); // @output 0
+    io:println((-9223372036854775807 - 1) & (-9223372036854775807 - 1)); // @output -9223372036854775808
```

Comment:
We usually use `int:MIN_VALUE` instead. Would that work for nBallerina?

Reply:
We don't support it yet.

Reply:
Ack. We need to add the `additive-expr` label.

Reply:
fixed